test/specs/products.js

```diff
@@ -162,40 +191,114 @@ describe('Products Page Tests', () => {
             await ProductsPage.logout();
         });
 
+        it("should display all the 6 products names sorted in descending order", async () => {
+            await ProductsPage.filterData('nameDescending')
+            const descendingProductsNames = await ProductsPage.productData('names');
+            const descendingExpectedProductsNames = Object.values(ProductNames)
+
+            expect(descendingProductsNames).toEqual(descendingExpectedProductsNames.reverse());
+            await ProductsPage.filterData('nameAscending')
+        });
+
+        it("should display all the 6 products names sorted in ascending order", async () => {
+            await ProductsPage.filterData('nameDescending')
+            await ProductsPage.filterData('nameAscending')
+            const ascendingProductsNames = await ProductsPage.productData('names');
+            const ascendingExpectedProductsNames = Object.values(ProductNames)
+
+            expect(ascendingProductsNames).toEqual(ascendingExpectedProductsNames);
+        });
+
+        it("should display all the 6 products prices sorted in descending order", async () => {
+            let descendingExpectedProductsPrices = [];
+            let descendingProductsPrices = [];
+            await ProductsPage.filterData('priceDescending')
+            const productPrice = await ProductsPage.productData('prices');
+            const dictValues = Object.values(ProductPrices)
+
+            for (let i = 0; i < productPrice.length; i++) {
+                descendingProductsPrices.push(Number(productPrice[i].replace("$", "")));
+            }
+
+            for (let i = 0; i < dictValues.length; i++) {
+                descendingExpectedProductsPrices.push(Number(dictValues[i].replace("$", "")));
+            }
+            descendingExpectedProductsPrices.sort((a, b) => b - a);
+
+            expect(descendingProductsPrices).toEqual(descendingExpectedProductsPrices);
+            await ProductsPage.filterData('nameAscending')
```

Comment:
why do you use this method here?

Reply:
it was used in order to restore the default change. after further investigation, I have observed that it wasn't needed and it has been removed